single_tech_samples/databricks/sample3_cluster_provisioning_and_data_access/10.configure-databricks-cli.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+
+DEPLOYMENT_PREFIX=${DEPLOYMENT_PREFIX:-}
+AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID:-}
+AZURE_RESOURCE_GROUP_NAME=${AZURE_RESOURCE_GROUP_NAME:-}
+AZURE_RESOURCE_GROUP_LOCATION=${AZURE_RESOURCE_GROUP_LOCATION:-}
+
+if [[ -z "$DEPLOYMENT_PREFIX" ]]; then
+    echo "No deployment prefix [DEPLOYMENT_PREFIX] specified."
+    exit 1
+fi
+if [[ -z "$AZURE_SUBSCRIPTION_ID" ]]; then
+    echo "No Azure subscription id [AZURE_SUBSCRIPTION_ID] specified."
+    exit 1
+fi
+if [[ -z "$AZURE_RESOURCE_GROUP_NAME" ]]; then
+    echo "No Azure resource group [AZURE_RESOURCE_GROUP_NAME] specified."
+    exit 1
+fi
+if [[ -z "$AZURE_RESOURCE_GROUP_LOCATION" ]]; then
+    echo "No Azure resource group [AZURE_RESOURCE_GROUP_LOCATION] specified."
+    exit 1
```

Comment:
Lets include the default value as `westus`
```suggestion
    echo "Default location will be set to -> westus"
    AZURE_RESOURCE_GROUP_LOCATION="westus"
```